apps/protocol-api/src/prisma/schema.prisma

```diff
@@ -19,16 +19,16 @@ enum GuildStatus {
 }
 
 model User {
```

Comment:
Nice -- did we also decide we wanted to do a `tipping_enabled` boolean so we have the option to use that to check enabled/disabled instead of the contract address only? Thinking in a situation where we've created the split, a user disables and then reenables. We could set the true/false of `tipping_enabled` so we can use the original contract address

Reply:
yes! great catch. sorry i missed that and thank you for doing a thorough review.

Reply:
Absolutely! Great job!